src/index.js

```diff
@@ -1,97 +1,117 @@
+/* eslint-disable no-use-before-define */
 import './style.css';
 
 const textInputField = document.querySelector('#text-input-feild');
 const addButton = document.querySelector('#add-button');
 const todosContainer = document.querySelector('.todos-container');
 
-const todos = [];
-
-addButton.addEventListener('click', () => {
-    if (textInputField.value.trim().length == '') {
-        return;
-    }
-
-    // create todo item
-    const todoItem = {
-        text: textInputField.value,
-        completed: false,
-    };
-
-    // add todo item to array
-    todos.push(todoItem);
-
-    // clear text input field
-    textInputField.value = '';
-
-    // render todo item
-    renderTodoItem(todoItem);
+let todos = [
+  {
+    text: 'gym',
+    completed: false,
+    index: 1,
+  },
+  {
+    text: 'car wash',
+    completed: false,
+    index: 2,
+  },
+];
+
+// Initial rendering of existing todos
+todos.forEach(renderTodoItem);
+
```

Comment:
- [ ] Great job implementing the required changes from the previous reviewer. I guessed you tampered something while trying to make some changes suggested to you. At this moment, at the page load, your todo list items do not render to the page. Kindly look into it and fix.

#### Your App
![your-app](https://github.com/AbuTalha3/Todo-list/assets/26597432/f6c1ec4c-b3f2-421c-acc7-9ec83a349d37)

### Template
![temp-app](https://github.com/AbuTalha3/Todo-list/assets/26597432/d911ffff-94ba-475f-8487-d4da8e115748)